pages/learn/deploy/delta.md

```diff
@@ -14,11 +14,13 @@ Once the delta (difference between the old and new image) is calculated, the dev
 
 These binary deltas save on the amount of data needed to be downloaded, reduce the storage space requirements on the device to perform an application update, and shorten the time when Docker is updating.
 
-## Enabling/Disabling delta updates
+## Deltas by default
+
+Devices running {{ $names.os.lower }} >= 2.47.1 or [ESR](esr) versions >= 2020.04 will have delta update behavior enabled by default. For devices running {{ $names.os.lower }} < 2.47.1, updating to >= 2.47.1 via a [self-service update][self-service-update] will automatically enable delta updates for the device.
 
-__Note__: Delta updates are enabled by default for devices running {{ $names.os.lower }} >= 2.47.1 or [ESR](esr) versions >= 2020.04
+## Enabling/Disabling delta updates
```

Comment:
Can you please revert the other changes and just reword this note like so?

```suggestion
__Note__: Delta updates are enabled by default for devices running balenaOS >= 2.47.1 or ESR versions >= 2020.04 and can be selectively disabled per device. Note that the value of the fleet configuration variable controlling deltas is ignored by these devices.
```

Later paragraphs need to be rewritten from the perspective of deltas being enabled by default to actually make sense so I think amending that note should be enough for now and I'll try to look into this separately.

Reply:
@tmigone Can you review/resolve this ^